### PR TITLE
K8SPS-327 - Fix GKE latest pipeline

### PR DESCRIPTION
**Pull request by tplavcic**

### CODE REVIEW

File: cloud/jenkins/ps-operator-latest-scheduler.groovy [hunk 1]
```diff
@@ -90,9 +86,9 @@ pipeline {
...
+                        build job: 'ps-operator-gke-latest', propagate: false, wait: true, parameters: [string(name: 'TEST_SUITE', value: "${TEST_SUITE}"),string(name: 'TEST_LIST', value: "${TEST_LIST}"),string(name: 'IGNORE_PREVIOUS_RUN', value: "${IGNORE_PREVIOUS_RUN}"),string(name: 'GIT_BRANCH', value: "${GIT_BRANCH}"),string(name: 'GIT_REPO', value: "${GIT_REPO}"),string(name: 'PLATFORM_VER', value: "${PLATFORM_VER}"),string(name: 'OPERATOR_IMAGE', value: "${OPERATOR_IMAGE}"),string(name: 'IMAGE_MYSQL', value: "${IMAGE_MYSQL}"),string(name: 'IMAGE_ORCHESTRATOR', value: "${IMAGE_ORCHESTRATOR}"),string(name: 'IMAGE_ROUTER', value: "${IMAGE_ROUTER}"),string(name: 'IMAGE_BACKUP', value: "${IMAGE_BACKUP}"),string(name: 'IMAGE_TOOLKIT', value: "${IMAGE_TOOLKIT}"),string(name: 'IMAGE_HAPROXY', value: "${IMAGE_HAPROXY}"),string(name: 'IMAGE_PMM_CLIENT', value: "${IMAGE_PMM_CLIENT}"),string(name: 'IMAGE_PMM_SERVER', value: "${IMAGE_PMM_CLIENT}")]
```
Comment (ptankov) [inline cloud/jenkins/ps-operator-latest-scheduler.groovy:89]: is there no better way than to write the same code 3 times?

Comment (tplavcic) [reply cloud/jenkins/ps-operator-latest-scheduler.groovy:89]: @panchal-yash please comment ^

Comment (panchal-yash) [reply cloud/jenkins/ps-operator-latest-scheduler.groovy:89]: Got it @ptankov will test with a loop to avoid multiple changes for the same thing.

Comment (panchal-yash) [reply cloud/jenkins/ps-operator-latest-scheduler.groovy:89]: https://github.com/panchal-yash/jenkins-pipelines/blob/ps-80-buil-add-pkgs-minitests-version-check/ps/jenkins/Testing/sample-scheduler.groovy

https://cloud.cd.percona.com/job/sample-scheduler/12/console
The for loop syntax works, will create a pr for that once this one is merge.

Comment (tplavcic) [reply cloud/jenkins/ps-operator-latest-scheduler.groovy:89]: @panchal-yash thank you!

**Review by panchal-yash — APPROVED** [review]

File: cloud/jenkins/ps-operator-latest-scheduler.groovy (hunk 1 above)
Comment (ptankov) [reply cloud/jenkins/ps-operator-latest-scheduler.groovy:89]: ok

**Review by ptankov — APPROVED** [review]

**Pull request merged**